Sources/BraveWallet/Crypto/Stores/AccountActivityStore.swift

```diff
@@ -7,6 +7,10 @@ import Foundation
 import BraveCore
 
 class AccountActivityStore: ObservableObject {
+  /// If we are displaying the selected account (ex. in `WalletPanelView`).
+  /// In some cases, we do not want to update the current account displayed
+  /// when the selected account changes (ex. when removing an account)
+  let isSelectedAccount: Bool
```

Comment:
The naming is a tad confusing here, since you do not check whether or not the account is actually the selected account when making the store. It also is only used for equality checks and to stop `selectedAccountChanged` from setting the account.

Perhaps we could replace this with something like `observeAccountUpdates` or something.

Reply:
Good point, updated in [f11c050](https://github.com/brave/brave-ios/pull/6293/commits/f11c0505aaa9936d8212619418d631b4895da1f1)